foundry.md

```diff
@@ -76,6 +82,12 @@ Foundry Cheat Codes
 module FOUNDRY-CHEAT-CODES
     imports EVM
     imports FOUNDRY-ACCOUNTS
+
+    configuration
+    <cheatcodes>
+        <expectRevert> false </expectRevert>
+        <expectedBytes> .ByteArray </expectedBytes>
+    </cheatcodes>
```

Comment:
The `expectRevert` cell seems to come from the [`Cheatcodes`](https://github.com/foundry-rs/foundry/blob/master/evm/src/executor/inspector/cheatcodes/mod.rs#L77) struct, but the `expectedBytes` is from the [`ExpectedRevert`](https://github.com/foundry-rs/foundry/blob/master/evm/src/executor/inspector/cheatcodes/expect.rs#L28) struct, and there is no representation for the depth. Perhaps encoding these different structs as different cells may be the way to go.